packages/cli/src/lib/generatePrismaClient.js

```diff
@@ -21,7 +21,7 @@ export const generatePrismaCommand = (schema) => {
   }
 
   return {
-    cmd: 'yarn prisma',
+    cmd: `node "${require.resolve('prisma/build/index.js')}"`,
```

Comment:
I think it would be better if we proxied the bin here too... just because prisma may decide to change this file, in which case we'd break! 

It would also mean bins are handled consistently

Reply:
Discussed with Dom, resolving for now